charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 9.0.1
+version: 9.1.1
```

Comment:
Shouldn't it have been 9.1.0? The patch version would be reset after a minor update, right?

Well, now it's too late...

Reply:
damn, should be 9.0.2.
but we'll leave it this way, as its already published.